src/components/Books.jsx

```diff
@@ -12,21 +12,14 @@ function Books() {
   const [cat, setCat] = useState('Fiction');
 
   return (
-    <div>
+    <div className="books">
       <BookList />
-      <div style={{
-        width: '100%',
-        background: 'black',
-        height: '2px',
-        marginTop: '10px',
-        marginBottom: '10px',
-      }}
-      />
-
-      <form>
-        <h1>ADD NEW BOOK</h1>
-        <div>
+      <form className="form">
+        <div className="form-line" />
+        <h1 className="form-title">ADD NEW BOOK</h1>
```

Comment:
- [ ] The horizontal line that separates the book cards and the form is not visible. Kindly add it and make it visible just as it is in the original design. 👍🏽

- [ ] Kindly style the Add Book button to match the original design. The button is expected to have a blue background and white text. Please modify that 👍🏽

Reply:
* This issue hasn't been fixed yet Kindly look into it ( the add button still looks different, and the spacing between the line that separates the cards ) 
* Also the vertical scroll bar kindly remove it as its not in the original design 
![image](https://github.com/amanuelbelete95/book-store-react/assets/52497246/2bd89ce7-faa4-4587-9b89-9f0cfb8d253d)
![image](https://github.com/amanuelbelete95/book-store-react/assets/52497246/b98ae99f-0f04-495b-9035-ef6401389f11)